examples/core/instancing/app.js

```diff
@@ -129,6 +131,30 @@ export default class AppAnimationLoop extends AnimationLoop {
       depthFunc: gl.LEQUAL
     });
 
+    setShaderHook('vs', {
+      signature: 'LUMAGL_pickColor(inout vec4 color)'
+    });
+
+    setShaderHook('fs', {
+      signature: 'LUMAGL_fragmentColor(inout vec4 color)'
+    });
+
+    setModuleInjection('vs', 'picking', {
+      shaderHook: 'LUMAGL_pickColor',
+      injection: 'picking_setPickingColor(color.rgb)'
+    });
+
+    setModuleInjection('fs', 'dirlight', {
+      shaderHook: 'LUMAGL_fragmentColor',
+      injection: 'color = dirlight_filterColor(color)'
+    });
+
+    setModuleInjection('fs', 'picking', {
+      shaderHook: 'LUMAGL_fragmentColor',
+      injection: 'color = picking_filterColor(color)',
+      priority: Number.POSITIVE_INFINITY
```

Comment:
Conventionally, lower priority number -> more important -> can overwrite the output by other modules (i.e. should be executed last).

Reply:
- I would say that probe.gl's use of the word "priority" is not very good, and I have gotten complaints that it is confusing (i.e. "lower" priority numbers have "higher" priority).
- In probe's case it hould really be changed to "logLevel".
- Maybe just avoid the word "priority" and use the word "order" here?

Reply:
"Priority" just means "coming first", so I'd find the "executed last" semantics extremely confusing. I prefer `priority` as a name, wouldn't be against `order` if everyone prefers that.

Reply:
Using `order` would save us from the "what does high priority mean" confusion.